<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="videoFrameHSV">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderUH">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>230</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderUS">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderUV">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>340</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelUS">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>260</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Saturation</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUV">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>310</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUH">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>200</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Hue</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLV">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>520</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLH">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>410</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Hue</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderLS">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>500</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelLS">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>470</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Saturation</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderLV">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>550</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hSliderLH">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>440</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="cameraCBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoFrameOri">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>57</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrated</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera_Calibration">
    <property name="title">
     <string>Camera Calibration</string>
    </property>
   </widget>
   <addaction name="menuCamera_Calibration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
